packages/zoe/src/contracts/callSpread.js

```diff
@@ -71,7 +71,9 @@ const start = zcf => {
 
   const terms = zcf.getTerms();
   const {
-    maths: { Collateral: collateralMath, Strike: strikeMath },
+    maths: { Collateral: collateralMath, Strike: strikeMath, Quote: quoteMath },
+    brands: { Strike: strikeBrand },
+    issuers: { Quote: quoteIssuer },
```

Comment:
Does this contract need to use the quoteIssuer?

Reply:
Not any more.